tests/insurance_test.go

```diff
@@ -42,7 +42,7 @@ func (c *ClientTests) TestInsuranceRetrieve() {
 	require.NoError(err)
 
 	assert.Equal(reflect.TypeOf(&easypost.Insurance{}), reflect.TypeOf(retrievedInsurance))
-	assert.Equal(insurance, retrievedInsurance)
+	assert.Equal(insurance.ID, retrievedInsurance.ID)
```

Comment:
But this test is for retrieve one insurance. Why should it use the retrieve all method?

Reply:
This is a trick we use in many of the library tests. In order to retrieve something you need an ID. So you have two options. Either create a new record or instead retrieve all (with a low page size), then snag the ID of the first returned item. If you look through the tests, you'll find we use this quite often.

In this case where a newly created object differs from the retrieved one for some reason, this could be a way around that.

@jchen293 can you create an internal ticket to investigate the discrepancy between objects/timestamps please? We definitely need to get to the bottom of this.